src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -334,12 +337,12 @@ def update_data(self):
 
     def get_next(self, sub_id):
         right = bisect_right(self.man_addr_list, sub_id)
-        if right == len(self.man_addr_list):
+        if right >= len(self.man_addr_list):
```

Comment:
Why is this change required?

Reply:
This is not required. It just make code more robust.

---
In reply to: [503552127](https://github.com/Azure/sonic-snmpagent/pull/164#discussion_r503552127) [](ancestors = 503552127)